Composer/packages/client/src/components/TestController/publishDialog.tsx

```diff
@@ -122,25 +157,60 @@ export const PublishLuisDialog: React.FC<IPublishLuisDialogProps> = (props) => {
     },
   };
 
-  const { formData, formErrors, hasErrors, updateField } = useForm(luisFormConfig, { validateOnMount: true });
+  const { formData, formErrors, hasErrors, updateField } = useForm(formConfig, { validateOnMount: true });
 
   const handlePublish = useCallback(
     (e) => {
       e.preventDefault();
       if (hasErrors) {
         return;
       }
-
-      onPublish(formData);
+      const newValue = formData;
```

Comment:
newValue and formData is the same object. if you don't want to change formData, can use Object.assign

Reply:
We delete some property from newValue at the next logic. so the newValue is different from formData.
```
      const newValue = formData;
      const subscriptionKey = formData.subscriptionKey;
      const qnaRegion = formData.qnaRegion;
      delete newValue.subscriptionKey;
      delete newValue.qnaRegion;
```